GOVERNANCE.md

```diff
@@ -0,0 +1,127 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release, Long Term Support (LTS) and
+Canary in the Gold Mine (CitGM) teams. It manages the release
+and long term support schedule and policies for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
+* Schedule for all releases.
+* Contribution policy for the Release repository.
+* Conduct guidelines for the Working group.
+* Administration of Long Term Support policy for all releases.
+
+For the current list of WG members, see the project README.md.
+
+## Collaborators
+
+The Release GitHub repository is maintained by the WG (all WG
+members are Collaborators for the Release repository) and additional
+Collaborators who are added by the WG on an ongoing basis.
+
+Individuals making significant and valuable contributions are made
+Collaborators and given commit-access to the project. These individuals
+are identified by the WG and their addition as Collaborators is discussed
+during the WG meeting.
+
+**Note**: If you make a significant contribution and are not considered for
+commit access, log an issue or contact a WG member directly and it will
```

Comment:
We have the same wording in the benchmarking WG, the post-mortem WG, Intl, the now inactive test WG used similar wording. 

Not a WG but one which is different is the community committee, which uses the participation rules.  Basically get involved and once you meet the participation rules you will be added.  The rules are:

```
In the case where an individual CC member -- within any three month period -- attends fewer than 25% of the regularly scheduled meetings, does not participate in CC discussions, and does not participate in CC votes, the member shall be automatically removed from the CC. The member may be invited to continue attending CC meetings as an observer.
```

In my opinion the existing wording is pretty generic in that "significant" can be determined by the WG and I don't think there have been problems with people asking to be added/getting refused.

So I'm open to a better suggestion if someone has one but I'm also fine with what we have already.